src/lib/plugin.js

```diff
@@ -27,6 +27,38 @@ function wait (ms) {
   }
 }
 
+function * webfingerToOptions (identifier) {
```

Comment:
Function name is not super-descriptive to me. Perhaps `resolveWebfingerCredentials` or `resolveWebfingerOptions` or something like that?